src/components/Personalization/createViewCollect.js

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import { isNonEmptyArray, assign } from "../../utils";
+
+export default ({ eventManager, mergeMeta }) => {
+  return ({ meta, xdm = {} }) => {
+    const { decisions = [] } = meta;
+    const data = { eventType: "display" };
```

Comment:
We should add this to the xdm as a known event type. Perhaps it should be more specific like "personalization.display"

Also, calling this variable "data" is confusing, because this is actually merged into the xdm. Consider "viewXdm"

Reply:
`xdm.eventType: "display"` has been added a while ago, this is not something that has been introduced in this PR. Konductor has logic around this `eventType` so we can't simply rename it.